examples/docs_snippets/docs_snippets/concepts/io_management/test_build_io_context.py

```diff
@@ -130,3 +130,16 @@ def test_input_context_partition_key():
 
     context = build_input_context()
     assert not context.has_partition_key
+
+
+def test_input_context_group_name():
```

Comment:
this file is tests for our documentation code snippets. These probably belong here instead https://github.com/dagster-io/dagster/blob/df768465caa56d0348f694556f8acdab3bb63e91/python_modules/dagster/dagster_tests/storage_tests/test_io_manager.py